lesson_30/app.test.js

```diff
@@ -0,0 +1,35 @@
+//  Homework lesson_30
+
+const appObj = require('./app');
+describe("test on isArray", () => {
+    it('should return "true"', () => {
+        expect(Array.isArray(appObj.checkArr)).toBe(true);
+    });
+});
+describe("test the math operations with array", () => {
+    it('should math sum elements return "number"', () => {
+        expect(typeof appObj.sumResoult).toBe("number");
+    });
+    it('should max element of array return "number"', () => {
+        expect(typeof appObj.maxValue).toBe("number");
+    });
+});
+describe("test clear, decrement and increment function from Acumulator", () => {
+    it('should typeof clear is "number"', () => {
+        expect(typeof appObj.clearFunc).toBe("number");
+    });
+    it('should typeof decrement is "number"', () => {
+        expect(typeof appObj.checkFunDecrement).toBe("number");
```

Comment:
why you don't check the value after firing function increment and decrement?

that's the main porpose of tests
tests are not for checking types or at least only types